release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,26 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 28 March 2024 - Fix - Catalog - Links in organization overview not working correctly [ID_39204]
+
+Links in the DataMiner Systems table of the organization overview will now correctly navigate to the right location.
+
+#### 28 March 2024 - Fix - Catalog - Typos in error when no DMS is found [ID_39232]
+
+When no DMS is found for an organization, the displayed error will now no longer contain any typos.
+
+#### 28 March 2024 - Enhancement - Admin - Adjusted visibility credits section organization overview [ID_39214]
+
+Regular members of an organization will now no longer be able to see the credits section of the organization overview.
+
+#### 28 March 2024 - Enhancement - Catalog - Filter on public and/or private items [ID_39026]
```

Comment:
The info about this filter feature should probably be added to the user guide as well, specifically here: https://docs.dataminer.services/user-guide/Cloud_Platform/Catalog/Looking_up_a_DataMiner_connector.html
Will you take care of this?

Reply:
That should indeed be in there.
The developer probably forgot to add it, i'll notify him to to make a PR for it.